opentelemetry-sdk/src/opentelemetry/sdk/environment_variables.py

```diff
@@ -369,3 +369,12 @@
 
 If both are set, :envvar:`OTEL_SERVICE_NAME` takes precedence.
 """
+
+OTEL_PYTHON_LOG_EMITTER_PROVIDER = "OTEL_PYTHON_LOG_EMITTER_PROVIDER"
```

Comment:
Should we add an underscore for this until it becomes stable?

Reply:
Done